app/src/main/java/com/hacka/presenteperfeito/feature/home/screen/util/GetInitials.kt

```diff
@@ -0,0 +1,8 @@
+package com.hacka.presenteperfeito.feature.home.screen.util
+
+fun getInitials(userName: String): String {
```

Comment:
Maninho poderia ficar no pacote Core>Common>Extensions, pois pode ser que seja utilizado essa logica fora do contexto da home.
```suggestion
fun String.getInitials(): String {
    return this
        .split(" ")
        .mapNotNull { it.firstOrNull()?.toString() }
        .joinToString("")
}
```